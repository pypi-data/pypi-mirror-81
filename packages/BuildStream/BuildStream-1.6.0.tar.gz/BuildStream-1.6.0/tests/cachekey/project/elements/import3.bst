kind: import
sources:
- kind: local
  path: files/local

config:
  target: /opt
