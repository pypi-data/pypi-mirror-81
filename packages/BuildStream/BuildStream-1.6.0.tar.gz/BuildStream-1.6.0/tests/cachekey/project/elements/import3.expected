76c6de73910086f2697ea748757a4f5e67c6e069131f28e3e54c380a7d6e8e18